official/utils/misc/distribution_utils.py

```diff
@@ -111,7 +110,8 @@ def get_distribution_strategy(distribution_strategy="default",
               all_reduce_alg))
     return tf.distribute.MirroredStrategy(
         devices=devices,
-        cross_device_ops=_MIRRORED_ALL_REDUCE_OPTIONS[all_reduce_alg])
+        cross_device_ops=_MIRRORED_ALL_REDUCE_OPTIONS[all_reduce_alg](
```

Comment:
wouldn't this fail is all_reduce_alg = None?

Reply:
I think you reviewed an outdated snapshot.  I refactored some of the changes while trying to remove some lint, PTAL.